### DISCUSSION TITLE
Blueprint nur für HA? Nicht für ioB

### DISCUSSION THREAD

**Discussion by vocaris9** [Blackymas/NSPanel_HA_Blueprint#138] [Q&A]
Hallo. Ich würde gerne die alternative Lösung für das NSPanel auch gerne verwenden. Ich habe aber kein HA sondern einen ioBroker.
Geht das damit auch?

**Comment by Blackymas** [comment]
Hallo,

das geht leider nicht außer ioBroker unterstützt yaml. Aber ich bin mir nicht sicher da das Blueprint für HA geschrieben wurde.

**Comment by MarcelW85** [comment]
Hallo,
Blueprints gibt es nur bei Home Assistant, daher wird dieses Projekt nicht unter ioBroker funktionieren.
